actions/jfrog-login/action.yml

```diff
@@ -14,28 +20,29 @@ outputs:
     value: ${{ steps.setup-jfrog-cli.outputs.oidc-user }}
   jfrog-url:
     description: "Base URL of Ledger's JFrog platform to be used for subsequent API/CLI operations."
-    value: "https://jfrog.ledgerlabs.net"
+    value: ${{ inputs.jf_url }}
   jfrog-domain:
     description: "Base domain of Ledger's JFrog platform to be used for subsequent API/CLI operations."
-    value: "jfrog.ledgerlabs.net"
+    value: ${{steps.expose-outputs.outputs.jfrog-domain }}
 
 runs:
   using: "composite"
   steps:
-    - uses: jfrog/setup-jfrog-cli@v4
-      id: setup-jfrog-cli
-      env:
-        JF_URL: "https://jfrog.ledgerlabs.net"
-      with:
-        oidc-provider-name: github-oidc
-        oidc-audience: jfrog-github
-    - name: Expose outputs as environment variables
-      run: | 
-        : Expose outputs as environment variables
-        echo "JFROG_USER=${{ steps.setup-jfrog-cli.outputs.oidc-user }}" >> $GITHUB_ENV
-        echo "JFROG_TOKEN=${{ steps.setup-jfrog-cli.outputs.oidc-token }}" >> $GITHUB_ENV
-        echo "JFROG_DOMAIN=${JFROG_DOMAIN}" >> $GITHUB_ENV
-        echo "JFROG_URL=https://${JFROG_DOMAIN}" >> $GITHUB_ENV
-      env:
-        JFROG_DOMAIN: jfrog.ledgerlabs.net
-      shell: bash
+  - uses: jfrog/setup-jfrog-cli@v4
+    id: setup-jfrog-cli
+    env:
+      JF_URL: ${{ inputs.jf_url }}
+    with:
+      oidc-provider-name: github-oidc
+      oidc-audience: jfrog-github
+  - name: Expose outputs as environment variables
+    id: expose-outputs
+    run: |
+      : Expose outputs as environment variables
+      JFROG_DOMAIN=$(echo "${{ inputs.jf_url }}" | awk -F[/:] '{print $4}')
+      echo "JFROG_DOMAIN=$JFROG_DOMAIN" >> $GITHUB_ENV
+      echo "::set-output name=jfrog-domain::$JFROG_DOMAIN"
```

Comment:
Why this change of way to output? _(please use the new way to output with the GITHUB variable, like for the env)_